cf-acceptance-tests/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN go get github.com/tools/godep
 RUN go get github.com/onsi/ginkgo/ginkgo
 
 # Install the cf CLI
-RUN wget -q -O cf.deb "https://cli.run.pivotal.io/stable?release=debian64&version=6.21.1&source=github-rel"
+RUN wget -q -O cf.deb "https://cli.run.pivotal.io/stable?release=debian64&version=6.25.0&source=github-rel"
 RUN dpkg -i cf.deb
```

Comment:
Would this be a good opportunity to combine these 2 lines and remove the downloaded deb after installation to save a little bit on the image size?